### PR TITLE
fixed missing otherLand label

### LINKED ISSUE #1590: Data Export: VariableSelect.tsx:89 Uncaught TypeError: Cannot destructure property 'key' of 'variable.props.label' as it is undefined.
Screenshot 2022-10-06 at 10 01 44

### PR DESCRIPTION
**Pull request by rfontanarosa**
close #1590

Schermata 2022-10-11 alle 11 54 50

### CODE REVIEW

File: src/test/sectionSpec/fraSpecs.ts [hunk 1]
```diff
@@ -160,6 +160,7 @@ export const FraSpecs: Record<string, SectionSpec> = {
                     colName: `${year}`,
                   })),
                 ],
+                labelKey: 'fraClass.otherLand',
```
Comment (minotogna) [inline src/test/sectionSpec/fraSpecs.ts]: this must be removed. labels are per cycle now.

File: src/client/pages/DataExport/utils/labelKeys.ts [hunk 2]
```diff
@@ -33,7 +33,7 @@ const columnLabelKeys: Record<string, string> = {
 export const getVariableLabelKey = (variable: string): string => {
```
Comment (minotogna) [inline src/client/pages/DataExport/utils/labelKeys.ts]: this should also be removed.

File: src/client/pages/DataExport/VariableSelect/VariableSelect.tsx [hunk 3]
```diff
@@ -103,8 +103,9 @@ const VariableSelect: React.FC<{ variables: Array<Row> }> = ({ variables }) => {
           <div className="export__form-section-variables">
             {variables.map((variable) => {
               const { variableName } = variable.props
-              const { key: labelKey, params: labelParams, prefix: labelPrefixKey } = variable.props.label
-              const label = getVariableLabelKey(labelKey)
+              const { key: labelKey, params: labelParams, prefix: labelPrefixKey } = variable.props.label ?? {}
+              const label = getVariableLabelKey(labelKey) ?? variableName
+
```
Comment (minotogna) [inline src/client/pages/DataExport/VariableSelect/VariableSelect.tsx]: here we should use Labels.getLabel

**Review by minotogna — CHANGES_REQUESTED** [review]
this actually must be updated, since labels have been migrated per cycle

**Review by minotogna — APPROVED** [review]

File: src/client/pages/DataExport/VariableSelect/VariableSelect.tsx [hunk 4]
```diff
@@ -52,13 +53,13 @@ const VariableSelect: React.FC<{ variables: Array<Row> }> = ({ variables }) => {
...
-          <h4>{i18n.t(Heading[assessmentName])}</h4>
+          <h4>{t(Heading[assessmentName])}</h4>
           <DefinitionLink
             className="margin-right-big"
             document="tad"
             anchor="1a"
-            title={`(${i18n.t('definition.definitionLabel')})`}
-            lang={i18n.i18n.language}
+            title={`(${t('definition.definitionLabel')})`}
+            lang={i18n.language}
```
Comment (sorja) [inline src/client/pages/DataExport/VariableSelect/VariableSelect.tsx:62]: we can extract language directly

**Review by sorja — APPROVED** [review]

**Pull request merged**